src/SearchResults/index.js

```diff
@@ -716,7 +716,11 @@ function extractNormalizedFacetValues(results, attribute) {
       results._state.getHierarchicalFacetByName(attribute);
     var currentRefinementSplit = unescapeFacetValue(
       results._state.getHierarchicalRefinement(attribute)[0] || ''
-    ).split(results._state._getHierarchicalFacetSeparator(hierarchicalFacet));
+    )
+      .split(results._state._getHierarchicalFacetSeparator(hierarchicalFacet))
+      .filter(function (name) {
+        return name !== hierarchicalFacet.rootPath;
+      });
```

Comment:
Maybe it already works, but not sure from just looking at this + the test, but:

- what if rootPath is multiple segments long, how does this comparison work? (I guess name is the path so it works)
- What if rootPath is repeated later in the path, like "root > something > root", will root be removed twice? Maybe we should return once it's been removed once or similar?

Reply:
[The docs say](https://www.algolia.com/doc/api-reference/widgets/hierarchical-menu/react-hooks/#widget-param-rootpath) "if the first level is not the root level" so I guess it implies `rootPath` only has one segment

But still, maybe what I should do instead is to `replace` the first occurence of `rootPath`, that would be safer

Reply:
yes, I think we should enforce that rootPath is at the beginning

Reply:
Yeah I added a condition with `startsWith`